### PR TITLE
Nighthawk/231 pitch issue categorization

### PR DESCRIPTION
**Pull request by hannahtuttle**
The labels that are added onto each repo needs to be updated before this is ready so that when when the classification labels are selected during a contest the labels update correctly.

### CODE REVIEW

File: src/templates/ReportForm.tsx [hunk 1]
```diff
@@ -66,7 +68,7 @@ const ReportForm = ({ data, location }) => {
   // state
   const [state, setState] = useState<ReportState>(initialState);
   const [attributedTo, setAttributedTo] = useState<string>("");
-  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```
Comment (captainmangoC4) [inline src/templates/ReportForm.tsx:71]: This was actually set to true by default for a purpose. We want to show a loading state until the finding details are fetched when we're editing findings. This same page is used for submitting and editing findings. If it's set to false by default, then we'll see the form populate with data for a new finding and then abruptly update when the finding details are fetched.

**Comment by captainmangoC4** [comment]
Let's prefix these labels with something like "classification-" so that they are easily parseable for de-duping purposes.

File: src/templates/ReportForm.tsx (hunk 1 above)
Comment (hannahtuttle) [reply src/templates/ReportForm.tsx:71]: Good catch, I changed that for testing something, I removed all my testing code, but I missed this! I will change it back now!

**Comment by SamusElderg** [comment]
@hannahtuttle & @captainmangoC4 can we close this one off? doing a bit of a cleanup of PRs in this repo

**Comment by SamusElderg** [comment]
Closing stale PR

**Pull request closed without merging**